moped-editor/src/queries/components.js

```diff
@@ -237,13 +240,21 @@ export const UPDATE_SIGNAL_COMPONENT = gql`
     $phaseId: Int
     $subphaseId: Int
     $completionDate: timestamptz
+    $componentTags: [moped_proj_component_tags_insert_input!]!
```

Comment:
glad you caught this! in the not too distant future we have the prospect of adding other maps-selected assets besides signals, so that might be the time to rework this a bit if needed.

Reply:
TIL about the graphql syntax `!` to denote non-nullability. Thanks!